llvm/test/CodeGen/X86/shift-combine.ll

```diff
@@ -787,3 +787,38 @@ define <4 x i32> @or_tree_with_mismatching_shifts_vec_i32(<4 x i32> %a, <4 x i32
   %r = or <4 x i32> %or.ab, %or.cd
   ret <4 x i32> %r
 }
+
+; FIXME: Reproducer for a DAGCombiner::combineShiftOfShiftedLogic
```

Comment:
Leftover FIXME

Reply:
Thanks!   (I actually realized that I had forgotten to remove this when I woke up this morning, but was not quick enough to fix it before you found it.)